l1-contracts/deploy-scripts/upgrade/EcosystemUpgrade.s.sol

```diff
@@ -163,16 +169,13 @@ contract EcosystemUpgrade is Script {
     struct BridgehubDeployedAddresses {
         address bridgehubImplementation;
         address ctmDeploymentTrackerImplementation;
-        address ctmDeploymentTrackerProxy;
```

Comment:
why get rid of this address? Tbh these scripts should be cleaned up first, there is quite a lot of code duplication

Reply:
> why get rid of this address?

Because after Gateway upgrade we do not redeploy it. It should be passed as input in ContractsConfig

>  Tbh these scripts should be cleaned up first, there is quite a lot of code duplication

Would appreciate some help here if you have the opportunity. Implementing upgrade flow after Gateway blocks V27